dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/DdRum.kt

```diff
@@ -29,7 +29,7 @@ interface DdRum {
     /**
      * Stop tracking the ongoing RUM Action.
      */
-    fun stopAction(context: Map<String, Any?>, timestampMs: Long): Unit
+    fun stopAction(type: String, name: String, context: Map<String, Any?>, timestampMs: Long): Unit
```

Comment:
cc @louiszawadzki 
Here's a change of API that was done in the native SDK that we need to reflect, it might need som additional work when you upgrade the dependency in the RN SDK